index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link href="styles.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Source+Serif+Pro:wght@400;700;900&display=swap" rel="stylesheet">
+  <title>Yiwen's resume</title>
+
+</head>
+
+<body>
+
+  <h1>Yiwen Yu</h1>
+  <h2>yyu01@mica.edu | 443-854-7551</h2>
```

Comment:
For an HTML resume, it also makes sense for your email address to be a link; you do that by using [`mailto`](https://css-tricks.com/snippets/html/mailto-links/):
```suggestion
  <h2><a href="mailto:yyu01@mica.edu">yyu01@mica.edu</a> | 443-854-7551</h2>
```